docs_src/microservices/application/ApplicationServices.md

```diff
@@ -2,97 +2,32 @@
-the provided data to an MQTT Broker as specified in the configuration.
-
-There are two primary triggers that have been included in the SDK that
-initiate the start of the function pipeline. First is via a POST HTTP
-Endpoint `/api/v1/trigger` with the EdgeX event data as the body.
-Second is the MessageBus subscription with connection details as
-specified in the configuration.
-
-Finally, data may be sent back to the message bus or HTTP response by
-calling `.complete()` on the context. If the trigger is HTTP, then it will
-be an HTTP Response. If the trigger is MessageBus, then it will be
-published to the configured host and topic.
-
-### Unsupported existing export service functions
-
-**Valid Event Check**--The first component in the pipe and filter,
-before the copier (described in the previous section) is a filter that
-can be optionally turned on or off by configuration. This filter is a
-general purpose data checking filter which assesses the device- or
-sensor-provided Event, with associated Readings, and ensures the data
-conforms to the ValueDescriptor associated with the Readings.
-
-- For example, if the data from a sensor is described by its metadata profile as adhering to a "Temperature" value descriptor of floating number type, with the value between -100° F and 200° F, but the data seen in the Event and Readings is not a floating point number, for example if the data in the reading is a word such as "cold," instead of a number, then the Event is rejected (no client receives the data) and no further processing is accomplished on the Event by the Export Distro service.
+One of the most common use cases for working with data that comes from the MessageBus is to filter data down to what is relevant for a given application and to format it. To help facilitate this, six primary functions  are included in the SDK. The first is the `FilterByProfileName` function which will remove events that do or do not match the configured `ProfileNames` and execution of the pipeline will cease if no event remains after filtering. The second is the `FilterByDeviceName` function which will remove events that do or do not match the configured `DeviceNames` and execution of the pipeline will cease if no event remains after filtering.  The third is the `FilterBySourceName` function which will remove events that do or do not match the configured `SourceNames` and execution of the pipeline will cease if no event remains after filtering. A `SourceName` is the name of the source (command or resource) that the Event was created from. The fourth is the `FilterByResourceName` which exhibits the same behavior as `DeviceNameFilter` except filtering the event's `Readings` on `ResourceName` instead of `DeviceName`. Execution of the pipeline will cease if no readings remain after filtering. The fifth and sixth provided functions in the SDK transform the data received to either XML or JSON by calling `XMLTransform` or `JSONTransform`.
```

Comment:
I would suggest putting this six primary functions in bullets so it is much easier to find and read.
Something like below
- `FilerByProfileName`: ....
- `FilterByDeviceName`: ...
- .....
-

Reply:
Great idea!

Reply:
Done